dockerfiles/base/build.sh

```diff
@@ -6,7 +6,10 @@
 # http://www.eclipse.org/legal/epl-v10.html
 
 IMAGE_NAME="eclipse/che-base"
-. $(cd "$(dirname "$0")"; pwd)/../build.include
+base_dir=$(cd "$(dirname "$0")"; pwd)
+. $base_dir/../build.include
 
 init "$@"
 build
+
+source $base_dir/all_tests.sh
```

Comment:
FYI bash can be used in all cli code (because we know for sure that bash is installed inside the docker image) while here we don't know if end-user has bash or not (and shebang is only /sh)